### DISCUSSION TITLE
Show ratio instead of percent on completed torrents

### DISCUSSION THREAD

**Discussion by dukobpa3** [openscopeproject/TrguiNG#140] [Ideas]
Just as idea
Could be useful with colorful progressbar
<img width="194" alt="Screenshot 2024-01-07 at 23 18 55" src="https://github.com/openscopeproject/TrguiNG/assets/1858880/721c6437-e255-4a9b-9a42-68afec04f561">

**Comment by qu1ck** [comment]
Yeah, without color differentiation or maybe a up/down arrow icon, it can be confusing how full progress bar can have 2.3584 number in it if you don't notice there is no `%` sign next to it.
